### ISSUE TITLE
Release more frequently

### ISSUE THREAD

**Issue by Alex-D** [trikoder/oauth2-bundle#59]
There is no tagged release since 1.1.0 in January 2019 and master is ~90 commits ahead with lot of new things.

Consider tag and publish more frequently to help up keep our `composer.json` clean (no `dev-master`) and a easier way to check changes through time.

Thank you for your work :)

**Comment by froozeify** [comment]
For the next release I think they wait to merge #18 then #21

**Comment by Alex-D** [comment]
These PR are features, so we can easily image a 1.2.0 now and a 1.3.0 with the #21 ^^

**Comment by spideyfusion** [comment]
We want to integrate https://github.com/trikoder/oauth2-bundle/pull/21 before tagging a new major release. There were a bunch of changes in-between which will require that we release it as `2.0.0`.

We'll show more diligence in the future. :-)

Our next major focus is going to be full Symfony 4.3 support.

**Comment by Alex-D** [comment]
Is there breaking changes?
If not, 2.0.0 is not required 
If yes, I am curious to know them since I'm on `dev-master` and it's hard to follow things via commit history.

Thank you and good luck! :)

**Comment by X-Coder264** [comment]
We are aware that there hasn't been a new release in a while and we will definitively try to be better in that regard in the future. 

Just a bit of a background though - first there was a breaking change in `SensioFrameworkExtraBundle` regarding PSR-7 wiring in the DIC and then we also wanted to focus on making the bundle basically feature complete (aka merge in #18 and #21). #18 was merged today and we expect that #21 will be merged soon as well.

We'll have to tag the new release as 2.0 as there have been some breaking changes that happened on the `master` branch since 1.1.0 (like adding some methods to the interface in #53 for example). In general we expect the upgrade path to 2.0 to be easy and in the ideal case (which should be the case for most apps using the bundle IMO) without any required code changes in your apps.

**Comment by Nyholm** [comment]
May I suggest to release a 2.0.0-alpha1 for us early adopters? We can release that now so people can test and find bugs.

**Comment by spideyfusion** [comment]
Done deal!

https://github.com/trikoder/oauth2-bundle/releases/tag/v2.0.0-beta1
